config/settings.yml

```
@@ -1,9 +1,12 @@
 app_name: 'compliance'
 path_prefix: '/api'
 old_path_prefix: '/r/insights/platform'
-platform_kafka_topic: 'platform.upload.compliance'
-platform_kafka_inventory_topic: 'platform.inventory.events'
-platform_kafka_validation_topic: 'platform.upload.validation'
+kafka_subscription_topics:
```

Should it be:
```suggestion
kafka_consumer_topics:
```
to match the `kafka_producer_topics`? That seems to make more sense to me when thinking of consumers and producers.

---

Changed.